src/react-super-select.js

```diff
@@ -761,11 +761,16 @@ class ReactSuperSelect extends React.Component {
         optionsCount = 0;
 
     if (!isArray(data)) {
-      forIn(data, (groupedOptions, heading) => {
-        options.push(this._getGroupHeadingMarkup(heading));
-        options = options.concat(this._getTemplatedOptions(groupedOptions, optionsCount));
-        optionsCount = optionsCount + groupedOptions.length;
-      });
+      if(!isEmpty(data)){
```

Comment:
@92arpitgoyal Thanks for filing this fix.  I am going to put in a simpler fix and release a new version today.  I can simply add an `&& !isEmpty(data)` check to line 763 rather than having two nested if/else blocks